kafka-ui-e2e-checks/src/main/java/utils/qaseIO/TestCaseGenerator.java

```diff
@@ -0,0 +1,70 @@
+package utils.qaseIO;
+
+import extensions.QaseExtension;
+import io.qase.api.QaseClient;
+import io.qase.client.ApiClient;
+import io.qase.client.api.CasesApi;
+import io.qase.client.model.Filters;
+import io.qase.client.model.TestCaseCreate;
+import io.qase.client.model.TestCaseListResponse;
+import lombok.SneakyThrows;
+import org.junit.platform.engine.support.descriptor.MethodSource;
+import org.junit.platform.launcher.TestExecutionListener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import utils.qaseIO.annotation.Suite;
+
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.Objects;
+
+import static io.qase.api.QaseClient.getConfig;
+
+public class TestCaseGenerator implements TestExecutionListener {
+
+    private static final Logger logger = LoggerFactory.getLogger(QaseExtension.class);
```

Comment:
can be replaced with `@Slf4j` annotation

Reply:
done